.gitignore

```diff
@@ -37,7 +37,8 @@
 nri-config-generator/dist/*
 nri-config-generator/vendor
 nri-config-generator/templates/*
-!nri-config-generator/templates/config.json.tmpl
+!nri-config-generator/templates/default/
+!nri-config-generator/templates/default/config.json.tmpl
```

Comment:
The template and templated are using the same tree because of the pipeline, right?

I would change the pipeline or whatever creates the file here so we can have a cleaner `.gitignore`.

But it is a NIT or an iteration for when we have to revamp the pipeline.